src/schema/v2/homeView/sectionTypes/Activity.ts

```diff
@@ -0,0 +1,29 @@
+import { GraphQLObjectType } from "graphql"
+import { pageable } from "relay-cursor-paging"
+import { ResolverContext } from "types/graphql"
+import { NotificationsConnection } from "../../notifications"
+import { NodeInterface } from "../../object_identification"
+import { HomeViewGenericSectionInterface } from "./GenericSectionInterface"
+import { HomeViewSectionTypeNames } from "./names"
+import { standardSectionFields } from "./GenericSectionInterface"
+
+export const HomeViewActivitySectionType = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: HomeViewSectionTypeNames.HomeViewSectionActivity,
+  description: "A user activity section in the home view",
+  interfaces: [HomeViewGenericSectionInterface, NodeInterface],
+  fields: {
+    ...standardSectionFields,
+
+    notificationsConnection: {
+      type: NotificationsConnection.type,
+
+      args: pageable({}),
+      resolve: (parent, ...rest) => {
+        return parent.resolver ? parent.resolver(parent, ...rest) : []
+      },
+    },
+  },
+})
```

Comment:
(Know this is still a draft so please ignore if this comment is premature)

It might be worthwhile to roll all of this up into a simple factory function. Aside from `fields` it seems as though each file below shares 90% of the same information. So something like 

```tsx
createHomeViewSection({
  name: ...
  description: ... 
  fields: {
    ...
  }
})
```
would be mostly sufficient, and we wouldn't need all of the duplicated file spread. 

And then a simple object could define things and roll it all up nicely, to be spread into the main type:

```tsx
const SECTIONS = [
  {
    name: ...
    description: ... 
    fields: ... 
  }
]

const SectionsTypes = SECTIONS.map(createHomeViewSection) 
``` 

This would probably be more maintainable later on through time, since the abundance of files creates the appearance of more underlying complexity than there really is.

Reply:
Thanks for the 👀. For now, I think I'll keep it as is. One thing I've noticed as devs start to work in the schema is the [distinction between section _instances_ and section _types_](https://docs.google.com/presentation/d/1j9yFZmkdelJ6y3YQiuJ-KaUTv0Pc9-ZwyTdrR5HrU9Q/edit#slide=id.g3024881a6d2_1_126) needed to be clearer. 

That's _one_ of the motivations for the current refactor: being able to see in a clear way, reified in the filesystem, that those are the most important first-class concepts when working with the new schema. Hoping to help people fall into the pit of success, and I think that added explicitness outweighs the benefit of any extra DRY-ness that factory would provide.

But not opposed to this idea if our thinking changes 👍🏽

Reply:
I think our disagreement falls along well-worn lines 😄  

On this question I'm [Team Sandi](https://sandimetz.com/blog/2016/1/20/the-wrong-abstraction) and not [the opposition](https://www.codewithjason.com/duplication-cheaper-wrong-abstraction/)

Also curious to hear what others think!

Reply:
Yah, i'm not a fan of abstraction generally, but this is a _great_ candidate for it.

Reply:
I'm interested in how an approach to DRY this up would materialize but I'd rather not introduce that abstraction in this PR. This organizational change is a lot easier to grok, especially for new contributors, the current single file and I'd rather lean on the clear explicitness than potentially introduce a difficult abstraction to work with.

Reply:
Aight, thanks all. Sounds good to me 🦅